.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,14 @@
-[Insert your comment here]
+[Describe this pull request. Link to relevant GitHub issues, if any.]
 
 ***
-Please make sure that these boxes are checked before closing your pull request.
-- [ ] Set milestone for this PR
-- [ ] Added a brief explanation about this change in CHANGELOG.md
-- [ ] Added unit test for this change
\ No newline at end of file
+
+**Before creating a pull request**
+
+- [ ] Document new methods and classes
+- [ ] Format code with `make format`
+
+**Before merging a pull request**
+
+- [ ] Set version target
```

Comment:
It would be good to mention how to set the version target. It would be something like "Set version target by selecting milestone on the right side"

Reply:
Should this basically just be deciding between whether this PR goes into the next minor or major release? If so, we should add that as well.

If it's supposed to wait until a major release, are we planning to keep some PRs open until the next major release is ready?

Reply:
No. In that case, we might have a branch for it. Actually, if we're planning to release a major version and minor version (I don't believe this will happen soon though), `master` would be for the major version release, `release-0.3` (0.3 for example) would be for the minor version release. We just merge the PR into the right branch.

Targeting version for a PR just can be done by selecting proper branch. One of the reasons of using milestone is to collect PRs and issues in each milestone pages per version to see what changes have been made in the release like [this](https://github.com/personalrobotics/aikido/milestone/4?closed=1).